src/cfclient/ui/tabs/LogClientTab.py

```diff
@@ -46,7 +46,22 @@
                                       "/ui/tabs/logClientTab.ui")[0]
 
 
-class LogClientTab(Tab, log_client_tab_class, logging.StreamHandler):
+class LogHandler(logging.StreamHandler):
+    def __init__(self, signal):
+        logging.StreamHandler.__init__(self)
```

Comment:
this is a scary way of doing it. I think recommended in Python3 would be super().__init__(), but in this particular case you should be able to just remove this line, since the ctor doesn't require any arguments.